manim/animation/transform.py

```diff
@@ -227,6 +227,30 @@ def interpolate(self, alpha: float) -> None:
 
 
 class ClockwiseTransform(Transform):
+    """Transforms Mobjects while moving them in Clockwise direction.
+
+    Examples
+    --------
+
+    .. manim:: ClockwiseTransform1
+        :quality: medium
+
+        class ClockwiseTransform1(Scene):
+            def construct(self):
+                self.play(ClockwiseTransform(Square(), Circle()))
+
+    .. manim:: ClockwiseTransform2
+        :quality: medium
+
+        class ClockwiseTransform2(Scene):
+            def construct(self):
+                t1 = Text("1").shift(UP)
+                t2 = Text("2").shift(DOWN)
+                self.add(t1, t2)
+                self.play(ClockwiseTransform(t1, t2))
```

Comment:
I like this example. Maybe it makes sense to show the effect of `Transform` vs. `ClockwiseTransform` side by side? If you don't feel like modifying this example further, it can be left as is.

Reply:
i'll change it